content/yaml-notification/publish-release-notes.md

```diff
@@ -112,15 +112,15 @@ You can use the **git log** command to generate release notes with commit massag
         echo "$notes" | tee release_notes.txt
 {{< /highlight >}}
 
-If you use this script localy it will generate release notes with all commits between tags e.x v1.0.0 and v2.0.0
+If you use this script localy, it will generate release notes with all the commits between tags e.g v1.0.0 and v2.0.0
 
 {{< highlight json "style=paraiso-dark">}}
     v2.0.0
     Fix bug A
     Fix bug B
 {{< /highlight >}}
 
-However, when using Codemagic, you will also need to configure the `CM_CLONE_DEPTH` environment variable. By default, this variable is set to clone only one commit for builds triggered by tags. To capture all commits between tags, e.x. v1.0.0 and v2.0.0, you should set`CM_CLONE_DEPTH` to a value greater than the number of commits between those tags (e.g., 10 or more). This adjustment will ensure that the script fetches and generates release notes from the desired commit range.
+However, when using Codemagic, you will also need to configure the `CM_CLONE_DEPTH` environment variable. By default, this variable is set to clone only one commit for builds triggered by tags. To capture all commits between tags, e.x. v1.0.0 and v2.0.0, you should set`CM_CLONE_DEPTH` to a value greater than the number of commits between those tags (e.g., 10 or more). This adjustment will ensure to capture all the commits.
```

Comment:
However, when using Codemagic, you will also need to configure the `CM_CLONE_DEPTH` environment variable. By default, this variable is set to clone only one commit for builds triggered by tags. To capture all commits between tags, e.g. v1.0.0 and v2.0.0, you should set`CM_CLONE_DEPTH` to a value greater than the number of commits between those tags (e.g. 10 or more). This adjustment will ensure to capture all the commits.